specification/storage/data-plane/Microsoft.StorageDataLake/stable/2018-11-09/DataLakeStorage.json

```diff
@@ -1684,6 +1670,244 @@
+            "headers": {
+              "Date": {
+                "description": "A UTC date/time value generated by the service that indicates the time at which the response was initiated.",
+                "type": "string",
+                "format": "date-time-rfc1123"
+              },
+              "ETag": {
+                "description": "An HTTP entity tag associated with the file or directory.",
+                "format": "etag",
+                "type": "string"
+              },
+              "Last-Modified": {
+                "description": "The data and time the file or directory was last modified.  Write operations on the file or directory update the last modified time.",
+                "format": "date-time-rfc1123",
+                "type": "string"
+              },
+              "Content-Length": {
+                "description": "The size of the resource in bytes.",
+                "type": "integer",
+                "format": "int64"
+              },
+              "x-ms-client-request-id": {
+                "x-ms-client-name": "ClientRequestId",
+                "type": "string",
+                "description": "If a client request id header is sent in the request, this header will be present in the response with the same value."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "description": "A server-generated UUID recorded in the analytics logs for troubleshooting and correlation.",
+                "pattern": "^[{(]?[0-9a-f]{8}[-]?([0-9a-f]{4}[-]?){3}[0-9a-f]{12}[)}]?$",
+                "type": "string"
+              },
+              "x-ms-version": {
+                "x-ms-client-name": "Version",
+                "description": "The version of the REST protocol used to process the request.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-client-request-id": {
+                "x-ms-client-name": "ClientRequestId",
+                "type": "string",
+                "description": "If a client request id header is sent in the request, this header will be present in the response with the same value."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "type": "string",
+                "description": "A server-generated UUID recorded in the analytics logs for troubleshooting and correlation."
+              },
+              "x-ms-version": {
+                "x-ms-client-name": "Version",
+                "type": "string",
+                "description": "The version of the REST protocol used to process the request."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageError"
+            }
+          }
+        }
+      },
+      "parameters": [
```

Comment:
Nit - move this to the top of the resource so it's easier to read?  Or just move it into the method since it's only being used once?

Reply:
I'm sure you'll find out, but I wonder how this generates since you're also including it in the uri path `"/{filesystem}/{path}?action=flush"`